skimage/io/manage_plugins.py

```diff
@@ -114,8 +111,10 @@ def _scan_plugins():
 
     for filename in config_files:
         name, meta_data = _parse_config_file(filename)
+        if 'provides' not in meta_data:
+            warnings.warn(f'file {filename} not recognized as a scikit-image io plugin, skipping.')
```

Comment:
should stack level be 3 or 4?

Reply:
It's unclear to me, but I also don't think it's important. In this case, the fix is not in the user code, but in the directory described in the error message.

Reply:
but this would be the user's plugin right?

Reply:
Yes and no. If you look at the original bug report, the issue was Windows putting .ini files in random folders, including the plugins folder...